install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml

```diff
@@ -28,6 +28,23 @@ spec:
       serviceAccountName: {{ $key }}-service-account
 {{- if $.Values.global.priorityClassName }}
       priorityClassName: "{{ $.Values.global.priorityClassName }}"
+{{- end }}
+{{- if eq $.Values.global.proxy.enableCoreDump true }}
+      initContainers:
+        - name: enable-core-dump
+{{- if contains "/" $.Values.global.proxy_init.image }}
+          image: "{{ $.Values.global.proxy_init.image }}"
+{{- else }}
+          image: "{{ $.Values.global.hub }}/{{ $.Values.global.proxy_init.image }}:{{ $.Values.global.tag }}"
+{{- end }}
+          imagePullPolicy: IfNotPresent
+          command:
+            - /bin/sh
+          args:
+            - -c
+            - sysctl -w kernel.core_pattern=/etc/istio/proxy/core.%e.%p.%t && ulimit -c unlimited
+          securityContext:
+            privileged: true
```

Comment:
:(. This ends up biting us all the time. Is the coredump enabled by default?

Reply:
No, it was turned off again. We've left it off by default for 1.0 due to security and disk space concerns. Envoy should produce the correct backtrace which should be enough for most cases. For 1.1 we'd like to make it possible to turn on by setting a pod label.

Reply:
@rshriram this review has stalled a bit.  I assume the heartburn is over `privileged: true`.  I've removed the approved flag as nobody has approved this PR (I think this PR was from the days of the bot setting approved from all folks in the OWNERS file).

I understand not wanting to expand the footprint of the securityContext usage - perhaps a conditional would make sense here?